ios/RNZendeskChat.m

```diff
@@ -87,12 +102,12 @@ @implementation RNZendeskChat
 
 // dismiss the current controller shown, if any
 RCT_EXPORT_METHOD(dismiss) {
-    dispatch_sync(dispatch_get_main_queue(), ^{
-        if(currentController != nil){
+    if(currentController != nil){
```

Comment:
To prevent a racing condition accessing `currentController` I would wrap everything in the main thread.

Reply:
Definitely better
changed here ca6588cabd7323b4bcccdc37c593b4695562d0cd